libs/model/src/lib/event.ts

```diff
@@ -11,12 +11,18 @@ import { Timestamp } from './timestamp';
 // Event types
 export type EventMeta = Meeting | Screening | Slate | unknown;
 
-export type AttendeeStatus = 'owner' | 'requesting' | 'accepted' | 'denied' | 'ended';
+export type AttendeeStatus = 'owner' | 'requesting' | 'accepted' | 'denied' | 'ended' | 'attended';
 export interface MeetingAttendee extends Person {
   uid: string,
   status: AttendeeStatus,
 }
 
+export interface ScreeningAttendee extends Person {
+  uid: string;
+  email: string;
+  status: AttendeeStatus;
```

Comment:
For now, we could just use 
```suggestion
  status: 'attended';
```
Until we add more features for it